src/components/comment/commentEntries.jsx

```diff
@@ -1,16 +1,28 @@
 import React from "react";
 import PropTypes from 'prop-types';
+import { format } from 'date-fns';
 import './style.scss';
-import LikeNumberGroup from '../../components/LikeNumberGroup';
-import HorizontalLine from '../../components/HorizontalLine';
+import LikeNumberGroup from '../LikeNumberGroup';
```

Comment:
@oyedejipeace I gave feedback on this specifically to you and @jsamchineme. Why are you finding it difficult to adopt to naming conventions ? Please make the required change.

Reply:
I am sorry, if it seems like I ignored the feedback. That wasn't a deleberate action. I guess I didn't fully  understand what was expected of me. I thought each of us was supposed to rename the files we named to follow the team's adopted naming conventions. I didn't realise I was supposed to do it for every file including the one's named by another team member. I
I have made all the neccessary changes and going forward I will make sure to seek clarity when giving an instruction.